en/resources/books-blogs.md

```diff
@@ -28,7 +28,7 @@ Manning Publications, October 2013.
 Packt Publishing, June 2013.
 - **[express - Middleware für node.js](http://www.amazon.de/express-Middleware-node-js-J%C3%B6rg-Krause/dp/1517281342/ref=sr_1_1?ie=UTF8&qid=1442001556&sr=8-1&keywords=1517281342)**,
 texxtoor, September 2015. In deutscher Sprache / in German language
-- **[JADE - die Template Engine für node.js](http://www.amazon.de/JADE-Die-Template-Engine-node-js/dp/1517282098/ref=sr_1_1?ie=UTF8&qid=1442001592&sr=8-1&keywords=1517282098)**,
+- **[Pug - die Template Engine für node.js](http://www.amazon.de/JADE-Die-Template-Engine-node-js/dp/1517282098/ref=sr_1_1?ie=UTF8&qid=1442001592&sr=8-1&keywords=1517282098)**,
```

Comment:
This needs to be reverted, because the title of the published book has not changed.